pkg/chain/ethereum/beacon.go

```diff
@@ -648,6 +644,17 @@ func (mrb *mockRandomBeacon) SubmitRelayEntry(
 	return nil
 }
 
+func (mrb *mockRandomBeacon) ReportRelayEntryTimeout() error {
+	mrb.currentRequestMutex.Lock()
+	defer mrb.currentRequestMutex.Unlock()
+
+	// Set the current request start block to nil, so that a new relay entry
+	// request can be begin.
+	mrb.currentRequestStartBlock = nil
+
+	return nil
+}
```

Comment:
Is the client in the current version calling this function?

Reply:
The client in the current version calls this function.
There is a log that appears every time the relay entry timeout happens:
```
2022-07-26T16:28:06.107+0200    WARN    keep-beacon      go-log@v1.0.5/log.go:180relay entry was not submitted on time, reporting timeout at block [2214]
ReportRelayEntryTimeout called!!!!!!!!!!!!!!!!!!!!!!!!
```

(I temporarily added my own log, just to be sure: ```ReportRelayEntryTimeout called!!!!!!!!!!!!!!!!!!!!!!!!```)